src/envoy/http/service_control/filter_fuzz_test.cc

```diff
@@ -177,14 +177,15 @@ DEFINE_PROTO_FUZZER(
         return &request;
       }));
 
-  // Fuzz the stream info.
-  TestStreamInfo stream_info = Envoy::Fuzz::fromStreamInfo(input.stream_info());
-  EXPECT_CALL(mock_decoder_callbacks, streamInfo())
-      .WillRepeatedly(ReturnRef(stream_info));
-  EXPECT_CALL(mock_encoder_callbacks, streamInfo())
-      .WillRepeatedly(ReturnRef(stream_info));
-
   try {
+    // Fuzz the stream info.
```

Comment:
Then doTest() should be outside of exception try catch block.  Envoy only throw exception on config flow,  not in data process flow.

Reply:
Done. Still kept it inside the `try/catch` scope to simplify filter setup.